database/seeders/admin.js

```diff
@@ -0,0 +1,7 @@
+const { admin1, admin2 } = require('../../constants/mocks');
```

Comment:
The admin created here is not a mock. It will be nice if we have it somewhere else outside the mock file.